web/docs/operators/transformations/rare.md

```diff
@@ -0,0 +1,37 @@
+# rare
+
+Shows the least common values. The dual to [`top`](top.md).
+
+## Synopsis
+
+```
+rare <field> [--count-field=<count-field>|-c <count-field>]
+```
+
+## Description
+
+Shows the least common values for a given field. For each field value, the result will be a singular event containing the value's count.
```

Comment:
Perhaps you can find a better way to phrase this sentence: `For each field value, the result will be a singular event containing the value's count.`

And/or you can add an example output to the example queries below.

Reply:
`For each event, a new event containing the count will be produced.` ?

Reply:
Or "for each unique value".